assets/js/util/svg.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Utility functions for SVGs.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * Gets the up/down SVG arrow for Google Chart tooltips.
+ *
+ * @since n.e.x.t
+ *
+ * @param {number} difference The difference which can be positive or negative.
+ * @return {string} SVG for the +/- difference.
+ */
+export const getChartDifferenceArrow = ( difference ) => {
+	if ( Number.isNaN( Number( difference ) ) ) {
+		return '';
+	}
+
+	return (
+		`<svg width="9" height="9" viewBox="0 0 10 10" fill="none" xmlns="http://www.w3.org/2000/svg" class="${ classnames( 'googlesitekit-change-arrow', {
+			'googlesitekit-change-arrow--up': difference > 0,
+			'googlesitekit-change-arrow--down': difference < 0,
+		} ) }">
+			<path d="M5.625 10L5.625 2.375L9.125 5.875L10 5L5 -1.76555e-07L-2.7055e-07 5L0.875 5.875L4.375 2.375L4.375 10L5.625 10Z" fill="currentColor" />
+		</svg>`
```

Comment:
@eugene-manuilov @asvinb – why not use the real `ChangeArrow` component and [return the string](https://github.com/WordPress/gutenberg/tree/trunk/packages/element#renderToString)?
```js
renderToString( <ChangeArrow { ...props } /> )
```
In this case, I'm not sure a utility is necessary for this.

Also, the change arrow color may be inverted depending on what it is used for (eg, bounce rate is red when difference is positive) which this doesn't seem to account for, but using the component would. If we're not considering this with the current implementation then this may already be a bug.

If it's too much effort we can handle this in a separate issue.

Reply:
@aaemnnosttv Thanks for that. I wasn't aware we actually had a `ChangeArrow` component. However, am thinking maybe we should keep the utility function which uses the component, since we are using the functionality in multiple places. What do you think?

Reply:
Sounds good for now 👍